BankAPILibrary/GiniBankAPILibrary/Sources/GiniBankAPILibrary/Documents/Configuration/Configuration.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Configuration.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+
+/**
+ Struct for configuration settings
+ */
+public struct Configuration: Decodable {
```

Comment:
@zladzeyka maybe we should use GiniConfiguration or GiniClientConfiguration?

Reply:
@zladzeyka thanks for your suggestion! For protocol I used the same naming as we have in payments. In case that we can't have simultaneously 2 objects with same name `ClientConfigurationService`, we can move forward with `ClientConfigurationServiceProtocol`